docs/UserGuide.md

```diff
@@ -304,29 +304,27 @@ Therefore, to avoid unintentionally losing any information while editing tags, w
   * `Employee List Index does not exist` - An invalid index was specified
   * `Employee List Index not specified` - No index was specified by the user
 
-### Restarting HRMate by clearing all existing records : `clear`
+### Resetting HRMate
+After playing with the different features in HRMate, you might feel that it's time to delete HRMate's sample employee 
+contacts and instead use your own. Rather than manually deleting every employee contact and leave record, here's a 
+quick and easy way to clear them in the app!
+#### Resetting HRMate by clearing all existing records : `clear`
```

Comment:
Not sure if this was intentional, but the introduction of commands in all other section uses ### as opposed to ####, and in this case, you split the original header "resetting HRMate" from the introduction of the clear command

Reply:
Changed heading size for consistency